packages/doc/docs/extensions/customize-extension.mdx

```diff
@@ -0,0 +1,282 @@
+
+   ```js
+   const { MyValidator } = require('./my-validator.js');
+
+   exports = MyValidator;
+   ```
+
+   You could also use `exports` to export multiple same type extensions.
+
+   ```js
+   const { MyValidator1 } = require('./my-validator1.js');
+   const { MyValidator2 } = require('./my-validator2.js');
+
+   exports = [MyValidator1, MyValidator2];
+   ```
+
+   You could also use `exports` to export multiple different type extensions.
+
+   ```js
+   const { MyValidator1 } = require('./my-validator1.js');
+   const { MyDataSource1 } = require('./my-data-source1.js');
+   const { MyDataSource2 } = require('./my-data-source2.js');
+
+   exports = {
+    validators: MyValidator1;
+    dataSources: [ MyDataSource1, MyDataSource2 ];
+   }
+   ```
+
+   </TabItem>
+
+   <TabItem value="typescript" label="NodeJS with Typescript">
+
+   :::info
+   Use NodeJS with ES6 is the same as Typescript.
+   :::
+
+   If you use `export` to export your customized extension class.
+
+   ```ts
+   export { MyValidator } from './my-validator.ts';
+   ```
+
+   You could also use `export` to export multiple same type or different type extensions.
+
+   ```ts
+   export { MyValidator1 } from './my-validator1';
+   export { MyValidator2 } from './my-validator2';
+   ```
+
+   But you could also use `export` with **key-value** type to export multiple different type extensions clearly.
+
+   ```ts
+   import { MyValidator1 } from './my-validator1';
+   import { MyDataSource1 } from './my-data-source1';
+   import { MyDataSource2 } from './my-data-source2';
+
+   export {
+    validators: MyValidator1;
+    dataSources: [ MyDataSource1, MyDataSource2 ];
+   }
+   ```
+
+   If you use `export default` to export your customized extension class.
+
+   ```ts
+   import { MyValidator } from './my-validator';
+
+   export default [MyValidator];
+   ```
+
+   You could also use `export default` to export multiple same type extensions by **array type**.
+
+   ```ts
+   import { MyValidator1 } from './my-validator1';
+   import { MyValidator2 } from './my-validator2';
+
+   export default [MyValidator1, MyValidator2];
+   ```
+
+   You could also use `export default` to export multiple different type extensions by **key-value type**.
+
+   ```ts
+   import { MyValidator1 } from './my-validator1';
+   import { MyDataSource1 } from './my-data-source1';
+   import { MyDataSource2 } from './my-data-source2';
+
+   export default {
+    validators: MyValidator1;
+    dataSources: [ MyDataSource1, MyDataSource2 ];
+   }
+   ```
+
+   </TabItem>
+
+   </Tabs>
+
+## Use your customized extensions
+
+You could use [npm link](https://docs.npmjs.com/cli/v8/commands/npm-link) to link extension package locally:
+
+```bash
+my-extension $ npm link
+```
+
+Or you could install local extension package by `npm i -g <extension-package-folder>`:
+
+```bash
+$ npm i -g /Users/<user-name>/Code/my-extension
+```
```

Comment:
We don't need to install packages globally.
```suggestion
```bash
$ npm i  /Users/<user-name>/Code/my-extension
```
```

Reply:
Thanks for the reminder, I have fixed it!